### PR TITLE
Adding images and adjusting layout

### PR DESCRIPTION
**Pull request by davidrnovak**
Added a whole bunch of images and adjusted some layout stuff.

### CODE REVIEW

File: src/static/images/doodles-hero.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="1189" height="575" viewBox="0 0 1189 575" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="logo--title" role="img">
+  <title id="logo--title">Teacher Moments</title>
```
Comment (enatario) [inline src/static/images/doodles-hero.svg]: Ooops this was my fault. Can you move the aria-labelledby, role and the whole title line to the `logo.njk` file?

File: src/_data/deep_features.json [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-      "title": "Multiplayer scenarios",
-      "description": "Real-time chat allows for participants to roleplay together as teachers and students.",
-       "image": {
-        "url": "http://placekitten.com/600/300",
-        "alt": "A kitten"
-      }
-    }
-  ]
+	"title": "Dive deeper into Teacher Moments",
+	"description": "Become a power user and explore even more features for creating and running your simulations.",
```
Comment (enatario) [inline src/_data/deep_features.json]: Just adjust the indent on all the changed json files

**Review by enatario — CHANGES_REQUESTED** [review]
Looks good! just a few small items

File: src/_data/deep_features.json (hunk 2 above)
Comment (davidrnovak) [reply src/_data/deep_features.json]: Added the requested changes. Let me know if the json spacing changes worked.

**Review by enatario — APPROVED** [review]
LGTM merge away!

**Pull request merged**